.linters/cpp/checkKeyword.py

```diff
@@ -58,6 +58,7 @@
     'KW_FIXED_STRING',
     'KW_TIMESTAMP',
     'KW_DATE',
+    'KW_TIME',
```

Comment:
I think it's better to make it unreserved.

Reply:
Emm.The other type is all reserved. Don't keep same?

Reply:
No. We will unreserve those if not appropriate.

Reply:
Reserved for the reduce/reduce conflict at now.